symfit/core/minimizers.py

```diff
@@ -3,6 +3,7 @@
 from collections import namedtuple, Counter
 
 from scipy.optimize import minimize, differential_evolution, basinhopping
+import scipy.optimize as so
```

Comment:
why not import in the style as the line above this one? Or, if that is starting to lead to namespace conflict we should rename everything to `so.something` (which I would prefer not to do)

Reply:
I need access to scipy.optimize.BFGS, which conflicts with our BFGS. I'll look for an alternative (`from so import BFGS as soBFGS`)